Devscord.DiscordFramework.UnitTests/Commands/RunnerOfIBotCommandMethodsTests/RunnerOfIBotCommandMethodsTestsService.cs

```diff
@@ -0,0 +1,65 @@
+﻿using Devscord.DiscordFramework.Framework.Commands.Builders;
+using Devscord.DiscordFramework.Framework.Commands.Services;
+using Devscord.DiscordFramework.Middlewares.Contexts;
+using Moq;
+using System.Collections.Generic;
+using System;
+using System.Threading.Tasks;
+using System.Text.RegularExpressions;
+using Devscord.DiscordFramework.Framework.Commands;
+using Devscord.DiscordFramework.Framework.Commands.Parsing.Models;
+using System.Reflection;
+
+namespace Devscord.DiscordFramework.UnitTests.Commands.RunnerOfIBotCommandMethodsTests
+{
+    class RunnerOfIBotCommandMethodsTestsService
```

Comment:
co ta nazwa ma wspólnego z tym co ta klasa robi?

Reply:
1. Użyłem tej nazwy ze względu na konwencje nazwy. Zauważ, że w projekcie jest klasa AntiSpamTestsService, która jest bardzo podobna do tej
2. Szczerze to nie wiem jak ją nazwać lepiej...